mungegithub/mungers/inactive-review-handler.go

```diff
@@ -0,0 +1,168 @@
+	features *features.Features
+}
+
+func init() {
+	h := &InactiveReviewHandler{}
+	RegisterMungerOrDie(h)
+}
+
+// Name is the name usable in --pr-mungers
+func (i *InactiveReviewHandler) Name() string { return "inactive-review-handler" }
+
+// RequiredFeatures is a slice of 'features' that must be provided
+func (i *InactiveReviewHandler) RequiredFeatures() []string {
+	return []string{features.RepoFeatureName, features.AliasesFeature}
+}
+
+// Initialize will initialize the munger
+func (i *InactiveReviewHandler) Initialize(config *github.Config, features *features.Features) error {
+	i.features = features
+	return nil
+}
+
+// EachLoop is called at the start of every munge loop
+func (i *InactiveReviewHandler) EachLoop() error { return nil }
+
+// AddFlags will add any request flags to the cobra `cmd`
+func (i *InactiveReviewHandler) AddFlags(cmd *cobra.Command, config *github.Config) {}
+
+// Determine whether the PR is active based on number of seconds
+// A PR is inactive if it is not being modified for more than one week
+func (i *InactiveReviewHandler) isPRActive(issueCreatedAt *time.Time, comments []*githubapi.IssueComment, reviewComments []*githubapi.PullRequestComment) bool {
+	lastHumanIssueUpdateTime, _ := findLastHumanIssueUpdateHelper(comments, issueCreatedAt)
+	lastHumanPullRequestUpdateTime, _ := findLastHumanPullRequestUpdateHelper(reviewComments, issueCreatedAt)
+
+	// if the PR is not being modified yet, regard it as active to prevent assigning a reviewer
+	if lastHumanIssueUpdateTime == nil && lastHumanPullRequestUpdateTime == nil {
+		return true
+	} else if lastHumanIssueUpdateTime == nil {
+		return time.Since(*lastHumanPullRequestUpdateTime).Hours() <= 7*24
+	} else if lastHumanPullRequestUpdateTime == nil {
+		return time.Since(*lastHumanIssueUpdateTime).Hours() <= 7*24
+	}
+
+	lastModif := lastHumanIssueUpdateTime
+
+	if lastHumanPullRequestUpdateTime.After(*lastModif) {
+		lastModif = lastHumanPullRequestUpdateTime
+	}
+
+	return time.Since(*lastModif).Hours() <= 7*24
+}
+
+// Suggest a new reviewer who is NOT any of the existing reviewers
+// (1) get all current assignees for the PR
+// (2) get potential owners of the PR using Blunderbuss algorithm (calling getPotentialOwners() function)
+// (3) filter out current assignees from the potential owners
+// (4) if there is no any new reviewer available, the bot will encourage the PR author to ping all existing assignees
+// (5) otherwise, select a new reviewer using Blunderbuss algorithm (calling selectMultipleOwners() function with number of assignees parameter of one)
+// Note: the munger will suggest a new reviewer when the PR currently does not have any reviewer
+//		 the munger will only run at most 5 times
+func (i *InactiveReviewHandler) suggestNewReviewer(issue *githubapi.Issue, potentialOwners weightMap, weightSum int64) string {
+	var newReviewer string
+
+	if len(potentialOwners) > 0 && issue.Assignees != nil && len(issue.Assignees) > 0 {
```

Comment:
`issue.Assignees != nil && len(issue.Assignees) > 0`. The first part of this condition is not needed.